chaoslib/litmus/ec2-terminate-by-id/lib/ec2-terminate-by-id.go

```diff
@@ -75,6 +75,8 @@ loop:
 				return errors.Errorf("ec2 instance failed to stop, err: %v", err)
 			}
 
+			common.SetTargets(id, "injected", "EC2 Instance ID", chaosDetails)
```

Comment:
```suggestion
			common.SetTargets(id, "injected", "EC2", chaosDetails)
```
Please check at other places as well.